buildSrc/src/main/kotlin/CodegenTestCommon.kt

```diff
@@ -48,7 +48,7 @@ private fun generateSmithyBuild(
                             ${it.extraCodegenConfig ?: ""}
                         },
                         "service": "${it.service}",
-                        "module": "${it.module}",
+                        "module": "${it.module.lowercase()}",
```

Comment:
This is still best effort. I suggest we keep this as-is unless we plan on coming up with a full mechanism to "massage" an arbitrary string into a valid crate name.

Reply:
Let me do a better function than lowercase. If we keep it as-is it generates a warning.